### ISSUE TITLE
SAMD Xibee Integration

### ISSUE THREAD

**Issue by Shikhar1998** [Cansat2019/Cansat2019_Electronics#13]
@dt1729 @sanujkul.

**Comment by sanujkul** [comment]
sercom4_uart.ino file that was uploaded  @SouravB97 was tested with Xbee for sending and receiving the string. 
However, @SouravB97 suggested sending raw values in packet than creating a string will reduce size of the packet. Nakul SIr still need to confirm this as he said, earlier, that Labview only accepts string.

**Comment by Shikhar1998** [comment]
Yes, discussed with @SouravB97. We will add both `int_32` and 'string' format temporarily, Until, we decide on whether to do ground processing.

**Comment by Shikhar1998** [comment]
#4.
